src/System.Net.Http.WinHttpHandler/src/System.Net.Http.WinHttpHandler.msbuild

```diff
@@ -27,7 +27,7 @@
     <CompileItem Include="$(CommonPath)\System\Net\Security\CertificateHelper.Windows.cs" />
     <CompileItem Include="$(CommonPath)\System\Runtime\ExceptionServices\ExceptionStackTrace.cs" />
     <CompileItem Include="$(CommonPath)\System\Threading\Tasks\RendezvousAwaitable.cs" />
-    <CompileItem Include="$(CommonPath)\System\Threading\Tasks\TaskToApm.cs" />
+    <CompileItem Include="$(CommonPath)\CoreLib\System\Threading\Tasks\TaskToApm.cs" />
```

Comment:
@maryamariyan i don't removed this line but substituted, do you agree?Now compile well. TYVM

Reply:
Should be fine. thanks @MarcoRossignoli.